# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/docpp/flex-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

The white space between ']' and ']' is necessary or else flex will discard
all closing braces and brackets of the corresponding line in docify.cc.
Patch required with flex version 2.5.31, not required with 2.5.4a.

--- doc++-3.4.10/src/docify.ll	2001-03-08 08:36:20.000000000 +0000
+++ doc++-3.4.10-patched/src/docify.ll	2004-01-13 22:12:27.000000000 +0000
@@ -314,7 +314,7 @@
 	{
 	if(argv[1][0] == '-')
 	    {
-	    fprintf(stderr, "Usage: docify [infile [outfile]]\n");
+	    fprintf(stderr, "Usage: docify [ infile [ outfile ] ]\n");
 	    return 0;
 	    }
 	ain = fopen(argv[1], "r");
